xdot/ui/actions.py

```diff
@@ -67,7 +67,7 @@ class NullAction(DragAction):
 
     # FIXME: The NullAction class is probably not the best place to hold this
     # sort mutable global state.
-    _tooltip_window = Gtk.Window(Gtk.WindowType.POPUP)
+    _tooltip_window = Gtk.Window.new(Gtk.WindowType.POPUP)
```

Comment:
I believe the old code works with older GObject, but I dont know the specifics.

If we need to support the older GObject, I guess the following might allow backwards compatibility

```py
if hasattr(Gtk.Window, 'new'):
    _tooltip_window = Gtk.Window.new(Gtk.WindowType.POPUP)
else:
    _tooltip_window = Gtk.Window(Gtk.WindowType.POPUP)
```

I'll be happy to explore this further if there is a need to support the older GObject.